agimus_demo_01_lfc_alone/README.md

```diff
@@ -9,14 +9,46 @@ Expected behavior: robot not moving, LFC is implementing a default PD+ controlle
 ```bash
 vcs import src < src/agimus-demos/agimus_demo_01_lfc_alone/dependencies.repos
 rosdep update --rosdistro $ROS_DISTRO
-rosdep install -y -i --from-paths src --rosdistro $ROS_DISTRO --skip-keys libfranka
+rosdep install -y -i \
```

Comment:
This one is very much specific to the installation inside of the docker image. This should work without skipping of the keys, as all of the repos required in the installation process that are missing in ROS binaries are pulled from git via `vcs-tool` which is two steps above. The only incorrect entries are `jrl-cmakekodules` and `catkin`